src/Core/Services/Implementations/OrganizationDomainService.cs

```diff
@@ -67,10 +67,12 @@ public async Task ValidateOrganizationsDomainAsync()
                 await _domainRepository.ReplaceAsync(domain);
                 await _eventService.LogOrganizationDomainEventAsync(domain, EventType.OrganizationDomain_NotVerified,
                     EventSystemUser.SSO);
+                _logger.LogInformation(Constants.BypassFiltersEventId, null,
```

Comment:
I think there's another override of `LogInformation` that will allow you not to pass in the `null` here.